test.sh

```diff
@@ -2,8 +2,14 @@
 
 set -eux
 
+EXTRA_ARGS=""
+
+if [ -n "${TARGET-}" ]; then
+    EXTRA_ARGS="--release --target=${TARGET}"
```

Comment:
Can we leave a comment why we need this here?

Reply:
Done.